conf/cmi/field.field.node.landing_page.field_content.yml

```diff
@@ -45,39 +46,55 @@ settings:
   handler: 'default:paragraph'
   handler_settings:
     target_bundles:
+      banner: banner
+      chart: chart
       columns: columns
-      list_of_links: list_of_links
       content_cards: content_cards
       content_liftup: content_liftup
-      banner: banner
-      liftup_with_image: liftup_with_image
+      event_list: event_list
       from_library: from_library
-      chart: chart
-      remote_video: remote_video
+      health_station_search: health_station_search
+      hearings: hearings
+      liftup_with_image: liftup_with_image
+      list_of_links: list_of_links
       map: map
       news_list: news_list
-      event_list: event_list
       phasing: phasing
       popular_services: popular_services
+      remote_video: remote_video
+      service_list: service_list
+      service_list_search: service_list_search
       target_group_links: target_group_links
       unit_accessibility_information: unit_accessibility_information
       unit_contact_card: unit_contact_card
       unit_search: unit_search
-      service_list: service_list
-      service_list_search: service_list_search
-      hearings: hearings
-      health_station_search: health_station_search
+      maternity_and_child_health_clini: maternity_and_child_health_clini
     negate: 0
     target_bundles_drag_drop:
```

Comment:
Remove all these additions under the ` target_bundles_drag_drop:` except `maternity_and_child_health_clini:` but give it `weight: 0`